src/main/java/tutorlink/studentpackage/Student.java

```diff
@@ -0,0 +1,25 @@
+package tutorlink.studentpackage;
+
+public class Student {
+    private String matricNumber;
+    private String name;
+    private double gpa;
+
+    public Student(String name, String matricNumber) {
+        this.name = name;
+        this.matricNumber = matricNumber;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getMatricNumber() {
+        return matricNumber;
+    }
+
+    @Override
+    public String toString() {
+        return this.name + ", matric no: " + this.matricNumber;
+    }
+}
```

Comment:
Student should override equals for ArrayList to properly delete student

Reply:
Sorry I don't understand how equals() is required to delete a Student from the StudentList.

Currently StudentList.deleteStudent(String matricNumber, String name) is implemented without requiring a comparison of Student objects.

Reply:
The current StudentList.deleteStudent create a new student object from name and id and call List<Student>.remove() to remove it, but the .remove() method needed the .equals() to be implemented, or else it will just check for reference (aka memory address) which will always be different despite two student object have the same value.

Of course, if you decided to change the logic to the current one we're using in DeleteStudentCommand, then it would work as normal again, but it would still come in handy to compare student object in a command in case a command ever need to

Reply:
Got it, will add the equals() method. Thanks for your clarification.